test/components/previews/pathogen_datepicker_preview/default.html.erb

```diff
@@ -0,0 +1 @@
+<%= pathogen_datepicker(id: "test_id", input_name: "test_input_name") %>
```

Comment:
Can you move all the previews in the `pathogen` directory, trying to keep all the Pathogen previews grouped so it cleans up the lookbook page a bit.